### ISSUE TITLE
How to use captions?

### ISSUE THREAD

**Issue by jkinley** [browniebroke/gatsby-image-gallery#529]
Any chance you can provide a simple example of how to use captions? It would be nice if the demo had an example.

**Comment by browniebroke** [comment]
Good point. It's a bit tricky to get a real caption in the demo as I'm sourcing the images from the filesystem, so I just have to work with files, and I don't have meta data attached to them. But we could get something like the filename to show how to do it.

For a more real world example, I'm using this library on [one of my website](https://github.com/browniebroke/carriere-web) where the images are coming from [DatoCMS](https://www.datocms.com/). I'm able to grab the caption for each image directly in the GraphQL query as you can see [here](https://github.com/browniebroke/carriere-web/blob/f752100430d6e65349cc36630905b88628e4ca4e/src/templates/gallery-page.js#L45). The website is open source, and if you want to play with it, the data is publicly readable.

Basically, each object in the array passed as `images` prop has a `caption` field which is used in the lightbox component. 

You can see it in action by clicking on the second image on this page, the caption is on the bottom left corner: https://www.carriere-alla.fr/photos/voutes/

<img width="1675" alt="image" src="https://user-images.githubusercontent.com/861044/93016068-59f06080-f5b6-11ea-8204-9e456bb850af.png">
